developer.env

```diff
@@ -0,0 +1 @@
+BACKEND_URL=https://wsc2019.api.ushahidi.io
```

Comment:
Could you remove this file? I think gitpod didn't delete the old version of the file so it shows up here

Reply:
deleted! :)